catroid/src/main/java/org/catrobat/catroid/backpack/BackpackScriptController.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2017 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.backpack;
+
+import org.catrobat.catroid.content.Script;
+import org.catrobat.catroid.content.bricks.Brick;
+
+import java.util.List;
+
+public final class BackpackScriptController {
+
+	public static Script pack(Script item) {
+		List<Brick> brickList = item.getBrickList();
+		return null;
+	}
+
+	public static Script packAndAddToVisibleBackpack(Script item) {
```

Comment:
Yes, the idea was to only add visible looks to the backpack (a list), because the invisible ones will be members of the objects that hold them (e.g. sprites).